README.md

```diff
@@ -33,9 +33,20 @@ This will be akin to the Web API that you built in the last sprint, only this ti
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 1. Explain the difference between `RDBMS` and `SQL`.
-1. Why do tables need a `primary key`?
-1. What is the name given to a table column that references the primary key on another table.
-1. What do we need in order to have a _many to many_ relationship between two tables.
+
+RDBMS refers to the style of database itself, whereas SQL refers to the query language used to execute queries on our tables.
```

Comment:
A RDBMS (relational database management software) is a way to manage your database, typically with a nice UI. For example, SQL Studio, DB Browser, etc.